src/app/page.tsx

```diff
@@ -1,52 +1,13 @@
-import HeroSection from './components/HeroSection';
 import HowItWorksSection from './components/HowItWorksSection';
-import { fetchInterviews } from '../data/fetchInterviewData';
-import TopicCard from './components/TopicCard';
-import { legalTopics } from '../config/topics.config';
-import ShowAllToggle from './components/ShowAllToggle';
-import { formSources } from '../config/formSources.config';
-
-export default async function TopicsPage() {
-  const interviewsResult = await fetchInterviews();
-  const { interviewsByTopic } = interviewsResult;
-
-  // Modify this to account for various jurisdictions
-  const serverKey = 'suffolkListLab';
-  const server = formSources.docassembleServers.find(
-    (server) => server.key === serverKey
-  );
-  const url = server ? server.url : 'https://apps.suffolklitlab.org';
-
-  const filteredTopics = legalTopics
-    .sort((a, b) => b.priority - a.priority)
-    .filter(
-      (topic, index) =>
-        topic.always_visible ||
-        (interviewsByTopic[topic.name] &&
-          interviewsByTopic[topic.name].length > 0)
-    );
+import TopicsSection from './components/TopicsSection';
+import HeroSection from './components/HeroSection';
 
+export default async function Page() {
   return (
     <div>
       <HeroSection />
       <HowItWorksSection />
-      <section id="topics">
-        <div className="container">
-          <h2>Browse court forms by category</h2>
-          <div className="row row-cols-1 row-cols-md-3 g-5 card-container">
-            {filteredTopics.map((topic, index) => (
-              <TopicCard
-                key={topic.codes[0]}
-                topic={topic}
-                interviews={interviewsByTopic[topic.name] || []}
-                index={index}
-                serverUrl={url}
-              />
-            ))}
-          </div>
-          {filteredTopics.length > 9 && <ShowAllToggle />}
-        </div>
-      </section>
+      <TopicsSection jurisdiction={'ma'} />
```

Comment:
How does jurisdiction correspond to the jurisdiction in the JSON endpoint, which uses the SALI/LMSS location descriptors? E.g., US states look like:

`NAM-US-US+MA` (could replace MA with any 2 letter state abbreviation).

Just want to check for consistency here--if this is jurisdiction, maybe we want to match the two formats, and map the jurisdiction code to a path. if it's just path, using the same variable name perhaps?

Reply:
I may just be using the term jurisdiction wrong.  In this case I'm using jurisdiction to denote which path/server we are looking at.  Rather than jurisdiction, what term should i use for the path (previously subdomain) ie /ma/ or /pa/.  I'm assuming they wont always be by state or I would use that.

I do have this set now so that path to server is 1 to 1, but I see now that we need to allow multiple servers' data within a path.  I will adjust and get those changes up

Reply:
I think "path" is fine. Just think it will be helpful, if the path isn't identical to the jurisdiction code, to not use the word "jurisdiction"